web/src/lib/components/StoreView.svelte

```diff
@@ -0,0 +1,117 @@
+<script lang="ts">
+    import PublicAuctionCard from "../components/PublicAuctionCard.svelte";
+    import type { Auction } from "../types/auction";
+    import type { User } from "../types/user";
+    import { type ILoader, getEntities, getStore, getProfile, ErrorHandler } from "../services/api";
+    import { onMount } from "svelte";
+    import { token, user } from "../stores";
+    import StoreNotFound from "./StoreNotFound.svelte";
+    import { afterNavigate } from '$app/navigation';
+    import type { IEntity } from '$lib/types/base';
+
+    export let storeName;
+    export let loader: ILoader;
+    export let entities: IEntity[] | null = null;
+
+    let auctions: Auction[] | null = null;
+    let store: User;
```

Comment:
if it's a user, we should call it user, not store.

Reply:
change to storeOwner